src/cljs/nr/gameboard.cljs

```diff
@@ -305,72 +304,6 @@
   (and (get-in @game-state [:options :spectatorhands])
        (not (not-spectator?))))
 
-(def ci-open "\u2664")
-(def ci-seperator "\u2665")
-(def ci-close "\u2666")
-
-(defn is-card-item [item]
-  (and (> (.indexOf item ci-seperator) -1)
-       (= 0 (.indexOf item ci-open))))
-
-(defn extract-card-info [item]
-  (if (is-card-item item)
-    [(.substring item 1 (.indexOf item ci-seperator))
-     (.substring item (inc (.indexOf item ci-seperator)) (dec (count item)))]))
-
-(defn create-span-impl [item]
-  (if (= "[hr]" item)
```

Comment:
Looks like this has been left out.

Reply:
huh, yes I must have missed that, will take a look

Reply:
Added this in b683f9da157eca86225e6a8cb46cee07f1013b7b